java/mtsj/core/src/main/resources/db/specific/h2/V0001__R001_Create_schema.sql

```diff
@@ -197,6 +197,6 @@ CREATE TABLE BinaryObject (
 -- *** RevInfo (Commit log for envers audit trail) ***
 CREATE TABLE RevInfo(
   id BIGINT NOT NULL GENERATED BY DEFAULT AS IDENTITY (START WITH 1),
-  timestamp BIGINT NOT NULL,
+  "timestamp" BIGINT NOT NULL,
```

Comment:
good catch. We need to document that change in release-notes and hints for upgrade.